P5/Source/Specs/unitDecl.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+    Copyright TEI Consortium.
+    Dual-licensed under CC-by and BSD2 licences
+    See the file COPYING.txt for details
+    $Date$
+    $Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="unitDecl">
+  <gloss versionDate="2018-07-18" xml:lang="en">unit declarations</gloss>
+  <desc versionDate="2018-07-18" xml:lang="en">provides information about units of measurement that are not members of the International System of Units.</desc>
+  <classes>
+    <memberOf key="model.encodingDescPart"/>
+    <memberOf key="att.canonical"/>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.global"/>
+  </classes>
+  <content>
+    <elementRef key="unitDef" minOccurs="1" maxOccurs="unbounded"/>
+  </content>
+  <exemplum xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+        <unitDecl>
+          <unitDef name="斤" xml:id="斤" type="weight">
```

Comment:
These `@name` attributes are not declared in the elementSpec of `<unitDef>` (thus breaking the build).